api/room/schema_query.py

```diff
@@ -280,7 +280,8 @@ def resolve_all_rooms(self, info, **kwargs):
     def resolve_all_remote_rooms(self, info, return_all=None):
         page_token = None
         filter = map_remote_room_location_to_filter()
-        location = 'all' if return_all else get_user_from_db().location
+        location = get_user_from_db().location
+        location = 'all' if not filter.get(location) or return_all else location
```

Comment:
@MCFrank16 your `else` condition seems like repetition. You can get rid of it.

Reply:
Let me check that @joshuaocero

Reply:
@joshuaocero is there any situation that an admin can view all remote rooms while his location is available in the map filter function?

Reply:
@MCFrank16 yes, if the admin passes the `return_all` field, they should be able to see all locations even if their location is available in the map filter.